src/components/navBar/navBar.module.scss

```diff
@@ -0,0 +1,39 @@
+.navBar {
+  width: 100vw;
+  font-size: 1rem;
```

Comment:
The `font-size` is by default `1rem` no need to mention it specifically.
https://github.com/Real-Dev-Squad/website-status/blob/6a25cf74187cf413b93418f5a0fab62c1545383d/src/styles/index.scss#L17